### PR TITLE
DOCS-2257 Polish WAAP - Overview

### PR DESCRIPTION
**Pull request by MaximBashurov**

### CODE REVIEW

File: mkdocs-4.8.yml [hunk 1]
```diff
@@ -209,11 +209,16 @@ nav:
     - Triggers:
       - Working with Triggers: user-guides/triggers/triggers.md
       - Trigger Examples: user-guides/triggers/trigger-examples.md
+    - Filtration Mode:
+      - Overview: admin-en/configure-wallarm-mode.md
+      - Configuration via the Rule: user-guides/rules/wallarm-mode-rule.md
+    - Details:
```
Comment (AnastasiaTWW) [inline mkdocs-4.8.yml:215]: try not to use in the documentation such general group names - it is unclear when users need to go there

File: docs/latest/about-wallarm/waap-overview.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+Wallarm's WAAP is a next-gen WAF (Web Application Firewall) supporting multiple API protocols, such as REST, SOAP, GraphQL and [others](../user-guides/search-and-filters/use-search.md#search-hits-by-api-protocols) and implying a deep packet inspection to provide a full coverage of [OWASP Top 10](https://owasp.org/www-project-top-ten/).
+
+![Attack by protocols](../images/user-guides/dashboard/api-protocols.png)
+
+Although oriented on the traffic processing in a cloud (cloud native WAAP), the solution supports a wide range of [deployment options](../installation/supported-deployment-options.md) enabling you to seamlessly integrate the platform with your environment without its modification.
+
+Wallarm's WAAP functions are provided within the Cloud Native WAAP [subscription plan](../about-wallarm/subscription-plans.md).
+
+Go deeper into Wallarm's WAAP peculiarities reading its [overview](https://www.wallarm.com/product/wallarm-waap) at the Wallarm official site.
```
Comment (AnastasiaTWW) [inline docs/latest/about-wallarm/waap-overview.md:15]: no links to wallarm.com, we have to well-described that in the documentation. we have discussed that the content from wallarm.com is very brief and is not smth to refer to

File: docs/latest/about-wallarm/waap-overview.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+To get monitoring or protection:
+
+1. Register account in the [US](https://us1.my.wallarm.com/signup) or [EU](https://my.wallarm.com/signup) Wallarm Cloud.
+1. [Deploy](../installation/supported-deployment-options.md) your node connected to the account.
+1. Set node's [filtration mode](../admin-en/configure-wallarm-mode.md).
+
+As soon as you did so, the node will immediately start acting. It will use the following tools by default:
+
+* Library **libproton** - a primary tool for detecting malicious requests. The library uses the component **proton.db** which determines different attack type signs as token sequences, for example: `union select` for the [SQL injection attack type](https://www.wallarm.com/what/structured-query-language-injection-sqli-part-1). If the request contains a token sequence matching the sequence from **proton.db**, this request is considered to be an attack of the corresponding type.
```
Comment (AnastasiaTWW) [inline docs/latest/about-wallarm/waap-overview.md:27]: why wallarm.com link? we have SQLi description in the documentation

File: docs/latest/about-wallarm/waap-overview.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
+# WAAP/WAF
+
+In Wallarm platform, WAAP (Web Application & API Protection) is a core protecting functionality provided by default and a set of tools for fine-tuning this protection. This article provides an overview of WAAP elements and tuning possibilities.
+
+## Cloud native WAAP
+
+Wallarm's WAAP is a next-gen WAF (Web Application Firewall) supporting multiple API protocols, such as REST, SOAP, GraphQL and [others](../user-guides/search-and-filters/use-search.md#search-hits-by-api-protocols) and implying a deep packet inspection to provide a full coverage of [OWASP Top 10](https://owasp.org/www-project-top-ten/).
```
Comment (AnastasiaTWW) [inline docs/latest/about-wallarm/waap-overview.md:7]: list all protocols in the overview, no need to refer users somewhere else

Comment (AnastasiaTWW) [reply docs/latest/about-wallarm/waap-overview.md:7]: now only owasp 10, we cover much more than only owasp 10 threats

File: docs/latest/about-wallarm/waap-overview.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+Wallarm's WAAP functions are provided within the Cloud Native WAAP [subscription plan](../about-wallarm/subscription-plans.md).
+
+## Protection by default
+
+Once you register your company account in the [US](https://us1.my.wallarm.com/signup) or [EU](https://my.wallarm.com/signup) Wallarm Cloud and [deploy](../installation/supported-deployment-options.md) the Wallarm protecting node connected to this account, the node will immediately start acting.
+
+![!Arch scheme1](../images/about-wallarm-waf/overview/filtering-node-cloud.png)
+
+For protection the node will use two built-in attack [protection libraries](protecting-against-attacks.md#tools-for-attack-detection) - libproton and libdetection. The node applies them sequentially: the first - **libproton** - to detect signs of the attacks of different types, the second - **libdetection** - for additional validation. Regular updates of both libraries from Wallarm and using double-check approach ensures no attack types will be missed and the level of false positives stays low.
```
[the remark this answers is not in the document]
Comment (AnastasiaTWW) [reply docs/latest/about-wallarm/waap-overview.md:23]: add 1 parapgrah briefly describing the approach and its value and then the list with library names and brief description of their tasks. btw as for libdetection - additional valudation of what?

File: docs/latest/attacks-vulns-list.md [hunk 6]
```diff
@@ -88,6 +88,71 @@ Additionally, the Wallarm filtering node employs several special attack and vuln
     <iframe width="1280" height="720" src="https://www.youtube.com/embed/27CBsTQUE-Q" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
     </div>
 
+## Attack types
+
+The Wallarm solution protects APIs, microservices and web applications from OWASP API Top 10 threats, API abuse and other automated threats.
```
Comment (AnastasiaTWW) [inline docs/latest/attacks-vulns-list.md:93]: there are 2 OWASP10 threat lists, for API and for web, mention them both

File: docs/latest/attacks-vulns-list.md [hunk 7]
```diff
@@ -88,6 +88,71 @@ Additionally, the Wallarm filtering node employs several special attack and vuln
     <iframe width="1280" height="720" src="https://www.youtube.com/embed/27CBsTQUE-Q" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
     </div>
 
+## Attack types
```
Comment (AnastasiaTWW) [inline docs/latest/attacks-vulns-list.md:91]: the document was very long and became even more longer and more difficult to get the right info

File: docs/latest/about-wallarm/waap-overview.md (hunk 4 above)
Comment (MaximBashurov) [reply docs/latest/about-wallarm/waap-overview.md:7]: > list all protocols in the overview, no need to refer users somewhere else

Removed a link - "other" protocols are displayed on the screenshot.

File: docs/latest/about-wallarm/waap-overview.md [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+
+The measures that the node will take when finding attacks depend on the selected **filtration mode**: on enabling the node, you can set it either for just `monitoring` and reporting in case of occurring attacks or to `block` the attacks once they occur.
+
+Being as simple as that in its root, the filtration mode defines how the node will react. However, it is more than that: you have a lot of [configuration possibilities](../admin-en/configure-wallarm-mode.md) to fine-tune the mode to correspond to your API structure and protection needs.
+
+## Protection fine-tuning
+
+As soon as you have your WAAP monitoring/protection working, you can fine-tune it by configuring:
+
```
[the remark this answers is not in the document]
Comment (MaximBashurov) [reply docs/latest/about-wallarm/waap-overview.md:32]: We will shorten it when working with the "Rules" branch.

File: docs/latest/attacks-vulns-list.md (hunk 7 above)
Comment (MaximBashurov) [reply docs/latest/attacks-vulns-list.md:91]: Created separate DOCS-2323 as we agreed.

File: docs/latest/attacks-vulns-list.md [hunk 9]
```diff
@@ -88,6 +88,71 @@ Additionally, the Wallarm filtering node employs several special attack and vuln
...
+To detect behavioral attacks, it is required to conduct syntax analysis of requests and correlation analysis of request number and time between requests.
+
+Correlation analysis is conducted when the threshold of request number sent to user authentication or resource file directory or a specific object URL is exceeded. Request number threshold should be set to reduce the risk of legitimate request blocking (for example, when the user inputs incorrect password to his account several times).
+
+* Correlation analysis is conducted by the Wallarm postanalytics module.
+* Comparison of the received request number and the threshold for the request number, and blocking of requests is conducted in the Wallarm Cloud.
+
+When behavioral attack is detected, request sources are blocked, namely the IP addresses the requests were sent from are added to the denylist.
+
+#### Configuration of behavioral attack protection
```
[the remark this answers is not in the document]
Comment (MaximBashurov) [reply docs/latest/attacks-vulns-list.md:143]: I do not see any of that here.

**Pull request closed without merging**